### DISCUSSION TITLE
How to export the model to TfLite??

### DISCUSSION THREAD

**Discussion by Bhateja-Akash** [facebookresearch/detectron2#3221] [Q&A]
Is there any suitable way for this conversion? If not, please provide some tips for a conversion like,
PyTorch -> Onnx -> Tensorflow -> TfLite.

In the PyTorch to Onnx conversion, what is the required output format? I have followed the documentation, but stuck at a step that outputs the following:
IndexError: too many indices for the tensor of dimension 2

**Comment by XBeg9** [comment]
Any recommendations on this? Is it possible to convert this model to TFLite?
